src/background.js

```diff
@@ -126,8 +126,25 @@ ipcMain.on('shutdown-finished', () => {
   app.quit()
 })
 
+// check if the second instance in locked
+const lock = app.requestSingleInstanceLock()
+
+if (!lock) {
+  app.quit()
+} else {
+  // if someone executes a second instace we should focus on the main window
+  app.on('second-instance', (event, commandLine, workingDirectory) => {
+    if (win) {
+      if (win.isMinimized()) win.restore()
+      win.focus()
```

Comment:
On Windows, it works that not a second instance is allowed. However, the `win.focus()` is not really working. It seems it tries to obtain the focus but it does not succeed (a sort of glitch).